assignments/F-02-93/js/components/Login.js

```diff
@@ -0,0 +1,75 @@
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
```

Comment:
move this code in `miniproject/pages/index.js` .

Reply:
Done